package.json

```diff
@@ -96,6 +96,7 @@
     "body-parser": "^1.20.1",
     "bunyan": "^1.8.15",
     "bunyan-format": "^0.2.1",
+    "cheerio": "^1.0.0-rc.12",
```

Comment:
The release candidate on v1 have been going for 5 years and get more downloads than the latest stable version. So not so risky.